app/views/docs/permissions.phtml

```diff
@@ -22,7 +22,7 @@
 
 <h2 id="server-integration"><a href="/docs/permissions#server-integration">Server Integration</a></h2>
 
-<p>A server or <a href="/docs/admin">admin integration</a> can be used for increased flexibility. When using a Server SDK in combination with the proper <a href="/docs/keys#scopes">API key scopes</a>, you can have any type of access to any of your project resources regardless of their permissions.</p>
+<p>Developers can use server integrations for increased flexibility. When using a Server SDK in combination with the proper <a href="/docs/keys#scopes">API key scopes</a>, you can have any type of access to any of your project resources regardless of their permissions.</p>
```

Comment:
Starting with "Developers" sounds a bit off, cold or remote. Can we start with "Server integration can be used for". We are all developers, it's not a 3rd party.

Reply:
Sure. I've made the change


**The conversation below is off topic**

I think we should consider if the docs should be written in a passive or active voice. There's a general distinction in grammar where:

> Server integrations can be used for increased flexibility.
This is a passive voice, which means there's no clear subject (persons) that we're addressing, which is generally considered weaker in writing, but acceptable for stuff like documentation. This is stating facts, but not suggesting you do something.

> You can use server integrations for increased flexibility.
This is considered an active voice. It's more instructive like you "should" do xxx or yyy to accomplish zzz. It's considered easier to read (at least what I was taught).

We should actually have a conversation about active vs passive voices in copy writing and docs... but that's a long conversation for later.